micall/tests/test_consensus_aligner.py

```diff
@@ -12,6 +14,29 @@
 from micall.utils.report_amino import ReportNucleotide
 
 
+@contextmanager
+def fixed_random_seed(seed):
+    original_state = random.getstate()
+    random.seed(seed)
+    try:
+        yield
+    finally:
+        random.setstate(original_state)
+
+
+def mutate_sequence(rate, seq):
+    def mutate(x):
+        if random.random() > rate:
```

Comment:
It's not a significant problem, but I think this should be `>=`, otherwise `mutate(0)` will sometimes mutate nucleotides.

Reply:
You are right, thanks.